INSTALL

```diff
@@ -132,7 +132,31 @@ Configure supports a number of useful options.  Run
 
 	Configure -h
 
-to get a listing.  See the F<Porting/Glossary> file for a complete list of
+to get a listing.
+
+Many Configure switches are expressed as C<key=value> pairs, for example:
+
+    -Dcc=clang
+
+Sometimes the value to be supplied for a switch is a string which contains
+spaces.  In that case, the value needs to be quoted so as to delimit that
+"shell word" from any following switch.  Example:
+
+    sh ./Configure -des \
+        -Doptimize="-O2 -pipe -fstack-protector -fno-strict-aliasing" \
+        -Dusedevel
+
```

Comment:
I'd advice to put `-des` last always, but I do not know why I would

Reply:
> I'd advice to put `-des` last always, but I do not know why I would

Well, I customarily start off with `-des -Dusedevel` ... but for the purpose of the example we need something after the quoted value.